### ISSUE TITLE
A11y_Code.visualstudio.com_Docs User Guide_Accessibility_Screenreader:Screenreader does not announce the text content that are provided in list structure.

### ISSUE THREAD

**Issue by SwatiTiwari-TCS** [microsoft/vscode-docs#3708]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

 

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS1.3.1;#MAS4.1.2;#Win10;#Visual Studio Code;#Narrator; #CAITestPass_May2020;

 

**Environment Details:**

Application Name: code.visualstudio.com

#URL:https://code.visualstudio.com

Windows Version: Windows10

Browser version: Microsoft Edge Version 84.0.516.1 (Official build) dev (64-bit)

 

**Repro Steps:**

1. Launch the application Url :https://code.visualstudio.com

2. Navigate to "Docs" link.
3. From left navigation, select "User Guide".
4. Navigate to "Accessibility" link.
5.Start screenreader and start navigating using Caps+Arrow key to list item text content below the zoom heading and observe the issue.

**Actual:**

Screen reader does not announce the text content that are provided in list structure while navigating below the zoom heading.

**This issue is Observe throughout the application for all the text content provided in list structure.**

**This issue is observe only in Narrator.**

**Expected:**
Screen reader should announce the text content provided in list structure as its announce the link only that provided between the text content.

**Attachment link for Reference:**
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/EbPc3AgFMUpOvFeIlsQTKL8BslnEevJYZYTeDxT4u0mf2g?e=HgHxXB

**Comment by sbatten** [comment]
We are converting markdown to display and Narrator fails to read the list items properly. NVDA does not have the issue. Narrator also fails to read the list items if looking at the markdown in github as well. We can probably just change how we present the information to avoid this, but either Narrator or the markdown conversion tool would need to be updated to fix this.

**Comment by sbatten** [comment]
closing this as an upstream issue since other markdown rendered lists on github cannot be read by narrator either.

**Comment by ManishaC1** [comment]
Verified the issue in below environment.
#URL: https://code.visualstudio.com/
Issue is fixed.
Github tags:#Closed

https://user-images.githubusercontent.com/105198219/217174811-0b33f06c-37a1-4e1a-901d-1da7e5a5e78c.mp4
